conf/packages.yaml

```diff
@@ -174,13 +170,13 @@ spkg:
   lcalc:
     descr: Michael Rubinstein's L-function calculator
     name: lcalc
-    url: http://pmmac03.math.uwaterloo.ca/~mrubinst/L_function_public/CODE/
+    url: ''
```

Comment:
hmm, it's weird that it is no longer online available. I searched a bit around, and maybe it's http://oto.math.uwaterloo.ca/~mrubinst/L_function_public/CODE/ now?